lib/bundle/skipper.rb

```diff
@@ -13,7 +13,17 @@ def skip?(entry, silent: false)
            formula[:official_tap] &&
            !formula[:bottled]
           reason = Hardware::CPU.arm? ? "Apple Silicon" : "Linux"
+
           puts Formatter.warning "Skipping #{entry.name} (no bottle for #{reason})" unless silent
+
+          # If the user wants to install things from a bundle even if no bottle exists
+          # we should let them.
+          if ENV["HOMEBREW_INSTALL_IF_BOTTLE_MISSING"] == "true"
+            message = "... but HOMEBREW_INSTALL_IF_BOTTLE_MISSING is set to true. Installing anyway!"
+            puts Formatter.warning message unless silent
+            return false
+          end
+
```

Comment:
```suggestion
          if ENV["HOMEBREW_BUNDLE_INSTALL_IF_BOTTLE_MISSING"]
            message = "...but HOMEBREW_BUNDLE_INSTALL_IF_BOTTLE_MISSING is set, so installing from source."
            puts Formatter.warning message unless silent
            return false
          end

```

Reply:
How about simplifying this so that it's just a single warning message either way. Something like

```suggestion
          if ENV["HOMEBREW_BUNDLE_INSTALL_IF_BOTTLE_MISSING"]
            puts Formatter.warning "Installing #{entry.name} from source" unless silent
            return false
          else
            puts Formatter.warning "Skipping #{entry.name} (no bottle for #{reason})" unless silent
          end

```